src/rules/prefer-destructured-store-props.ts

```diff
@@ -0,0 +1,340 @@
+import type { TSESTree } from "@typescript-eslint/types"
+import { getPropertyName } from "eslint-utils"
+import type { AST } from "svelte-eslint-parser"
+import type { SuggestionReportDescriptor } from "../types"
+import { createRule } from "../utils"
+import {
+  findAttribute,
+  isExpressionIdentifier,
+  findVariable,
+} from "../utils/ast-utils"
+
+type StoreMemberExpression = TSESTree.MemberExpression & {
+  object: TSESTree.Identifier & { name: string }
+}
+
+const keywords = new Set([
```

Comment:
Would something like [`esutils`](https://www.npmjs.com/package/esutils#keyword) be useful here since it's less likely to drift out of sync?

Reply:
Thank you for checking this PR. I think it would be great to replace with esutils as you suggested! I will change this PR.